snmp/assets/dashboards/troubleshooting_dashboard.json

```diff
@@ -0,0 +1,2267 @@
+                                            "data_source": "metrics",
+                                            "name": "query1",
+                                            "query": "sum:datadog.agent.started{$Host} by {version}.as_count()"
+                                        }
+                                    ],
+                                    "response_format": "timeseries",
+                                    "style": {
+                                        "line_type": "solid",
+                                        "line_width": "normal",
+                                        "palette": "blue"
+                                    }
+                                }
+                            ],
+                            "show_legend": true,
+                            "title": "Agent Restarts",
+                            "title_align": "left",
+                            "title_size": "16",
+                            "type": "timeseries"
+                        },
+                        "id": 5753872191409390,
+                        "layout": {
+                            "height": 2,
+                            "width": 4,
+                            "x": 8,
+                            "y": 2
+                        }
+                    }
+                ]
+            },
+            "id": 7775029902313268,
+            "layout": {
+                "height": 5,
+                "width": 12,
+                "x": 0,
+                "y": 5
+            }
+        },
+        {
+            "definition": {
+                "background_color": "green",
+                "layout_type": "ordered",
+                "show_title": true,
+                "title": "SNMP",
+                "type": "group",
+                "widgets": [
+                    {
+                        "definition": {
+                            "autoscale": true,
+                            "precision": 2,
+                            "requests": [
+                                {
+                                    "formulas": [
+                                        {
+                                            "formula": "query1"
+                                        }
+                                    ],
+                                    "queries": [
+                                        {
+                                            "aggregator": "last",
+                                            "data_source": "metrics",
+                                            "name": "query1",
+                                            "query": "sum:snmp.device.reachable{agent_host:$Host.value}"
+                                        }
+                                    ],
+                                    "response_format": "scalar"
+                                }
+                            ],
+                            "title": "Devices reachable",
```

Comment:
I tend to think that timeseries are more useful than single query value since it's easier to spot issues over time.

Example for device reachable: 
![image](https://github.com/DataDog/integrations-core/assets/49917914/21dc27e7-96f7-4d32-9ddb-f0bcae6d49ba)

Similar for other cases like devices monitored, unreachable, etc.

Reply:
WDYT of time series backgrounds on these widgets ? It gives you info about evolution across time and you can quickly see the current value